openapi/api_session.template.yaml

```
@@ -121,12 +121,11 @@ paths:
         This is intended to use with ioweb-profile lock functionality.
       parameters:
         - $ref: "#/parameters/FiscalCode"
-        - $ref: '#/parameters/Token'
+        - $ref: "#/parameters/Token"
         - in: body
-          name: unlockcode
-          required: true
+          name: body
```

I think this parameter is required
```suggestion
          name: body
          required: true
```

---

Right, done in [[#IOPID-858] fix based on comments](https://github.com/pagopa/io-backend/pull/1060/commits/238a2e6b0a6fe45b06207dc69d4ad5917805b39e)